src/docs/sphinx/t_conduit_docs_blueprint_examples_out.txt

```diff
@@ -79,7 +79,7 @@ transformed to 'interleaved' mcarray
 }
 
 mem_spaces: 
-  0x7fc814e027e0: 
+  0x7f8890702090: 
```

Comment:
Excluding this from the log would also be ideal as it will likely change each update. The easiest workaround probably involves just verifying the addresses with assert statements instead of printing them out.

Reply:
For this case, it is used in the docs to demo the differences between how the memory is alloced by the nodes.
I don't think we can get away with hiding it.

(https://llnl-conduit.readthedocs.io/en/latest/blueprint.html#top-level-blueprint-interface)


Side Note: Not too happy with the schema formatting in many of these examples either, but thats a side effect of a prior change.